### PR TITLE
Fix Y axis plot label control

### PR DESCRIPTION
**Pull request by charlesh88**
Addresses https://github.com/nasa/openmct/issues/2464.

### Changes
- Now uses writing-mode: vertical-lr instead of transform for vertical text;
- Text overflow: hidden prevents scrollbars;
- Select displayed on hover over Y axis area;
- Dynamically adds icon-gear when Y axis can be configured;

Screen Shot 2019-08-30 at 5 12 08 PM

### Todos
- @deeptailor to wire up static label to dynamically update when user makes a change via the Y label select

### CODE REVIEW

**Comment by deeptailor** [comment]
@charlesh88 - the label is dynamic now. There is a conflict in _legacy-plots.scss, could you take a look?

**Comment by deeptailor** [comment]
I also added a fix for the export png and jpg buttons. We are targeting the plot div instead of the parent div, removing the need to hide and unhide the buttons.

**Comment by charlesh88** [comment]
> I also added a fix for the export png and jpg buttons. We are targeting the plot div instead of the parent div, removing the need to hide and unhide the buttons.

That's awesome, thanks.

File: src/plugins/plot/src/telemetry/PlotController.js [hunk 1]
```diff
@@ -277,22 +277,18 @@ define([
...
      */
     PlotController.prototype.exportJPG = function () {
-        this.hideExportButtons = true;
-        this.exportImageService.exportJPG(this.$element[0], 'plot.jpg', 'export-plot')
-            .finally(function () {
-                this.hideExportButtons = false;
-            }.bind(this));
+        var plotElement = this.$element.children()[1];
+
+        this.exportImageService.exportJPG(plotElement, 'plot.jpg', 'export-plot');
```
Comment (akhenry) [inline src/plugins/plot/src/telemetry/PlotController.js]: Does this still export the plot legend?

Comment (deeptailor) [reply src/plugins/plot/src/telemetry/PlotController.js]: Yes, the only div its not capturing is 
` <div class="c-control-bar">`

**Comment by akhenry** [comment]
## Reviewer Checklist

* Changes appear to address issue? Y
* Appropriate unit tests included? N/A
* Code style and in-line documentation are appropriate? Y
* Commit messages meet standards? Y

**Pull request merged**